controller_manager/src/controller_manager.cpp

```diff
@@ -75,6 +75,57 @@ bool controller_name_compare(const controller_manager::ControllerSpec & a, const
   return a.info.name == name;
 }
 
+/// Checks is a command interface belongs to a controller based on its prefix.
+/**
+ * A command interface can be provided by an controller in which case is called "reference"
+ * interface.
+ * This means that the @interface_name start with the name of a controller.
+ *
+ * \param[in] interface_name to be found in the map.
+ * \param[in] controllers list of controllers to compare their names to interface's prefix.
+ * \param[out] interface_controller_name name of the controller interface belongs to.
+ * \return true if interface has a controller name as prefix, false otherwise.
+ */
+bool command_interface_is_reference_interface_of_controller(
+  const std::string interface_name,
+  const std::vector<controller_manager::ControllerSpec> & controllers,
+  controller_manager::ControllersListIterator & following_controller_it)
+{
+  auto split_pos = interface_name.find_first_of('/');
+  if (split_pos == std::string::npos)  // '/' exist in the string (should be always false)
+  {
+    RCLCPP_FATAL(
+      rclcpp::get_logger("ControllerManager::utils"),
+      "Character '/', was not find in the interface name '%s'. This should never happen. "
+      "Stop the controller manager immediately and restart it.",
+      interface_name.c_str());
+    return false;
+    // TODO(destol): Should we here throw an error? I don't like the idea, but we should
```

Comment:
I would rather get the answer to this question so we can delete the comment.

Reply:
You are right. I think there should be a throw here instead of a boolean return.

The error message says it best: 

>       "Stop the controller manager immediately and restart it.",
which is somewhat equivalent to [HCF](https://en.wikipedia.org/wiki/Halt_and_Catch_Fire_(computing))

Reply:
Added `runtime_error` throw